src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/sdl_rpc_plugin.cc

```diff
@@ -131,14 +131,19 @@ void SDLRPCPlugin::RevertResumption(application_manager::Application& app) {
   pending_resumption_handler_->OnResumptionRevert();
 
   if (application_manager_->IsAppSubscribedForWayPoints(app)) {
-    application_manager_->UnsubscribeAppFromWayPoints(app.app_id());
-    if (!application_manager_->IsAnyAppSubscribedForWayPoints()) {
+    std::set<uint32_t> subscribed_apps =
+        application_manager_->GetAppsSubscribedForWayPoints();
+    bool send_unsubscribe = subscribed_apps.size() <= 1 &&
```

Comment:
@JackLivio const bool?

Reply:
@JackLivio do we really need to check "less than one"?
```suggestion
    bool send_unsubscribe = subscribed_apps.size() == 1 &&
```

Reply:
@AKalinich-Luxoft Updated the modifiers, but I would suggest keeping <= for "safety".